ci/ray_ci/doc_builder_container.py

```diff
@@ -4,12 +4,13 @@
 class DocBuilderContainer(Container):
     def __init__(self) -> None:
         super().__init__("docbuild")
-        self.install_ray()
+        self.install_ray(build_type="doc")
 
     def run(self) -> None:
         self.run_script(
             [
                 "cd doc",
+                "pip install -r requirements-doc.txt",
```

Comment:
this should have been installed in [1]?

Reply:
One would have thought so. But in actuality it seems like there were _old versions_ of sphinx and other dependencies that were being called when building the docs. IMO we should take a closer look at the CI system after we have this merged, because it's _really_ hard as someone who hasn't worked on the CI system to understand at the moment.

Reply:
probably because doc.build.Dockerfile was installed with `-U --ignore-installed`?